packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -331,4 +296,24 @@ export class NavBase extends React.Component<INavProps, INavState> implements IN
 
     return false;
   }
+
+  private _isGroupExpanded(group: INavLinkGroup): boolean {
+    if (group.name && this.state.isGroupCollapsed.hasOwnProperty(group.name)) {
+      return !this.state.isGroupCollapsed[group.name];
+    }
+    if (group.collapseByDefault !== undefined) {
```

Comment:
you can just return !group.collapseByDefault, although this way is a little more clear.

Reply:
In order to have properly controlled collapsed state on a group, I wanted this to fall through so the next author would have an obvious place to write the next `if`. Otherwise, you're spot-on.